simpleclient/src/main/java/io/prometheus/client/Summary.java

```diff
@@ -1,18 +1,20 @@
 package io.prometheus.client;
 
-import java.util.ArrayList;
-import java.util.List;
-import java.util.Map;
+import io.prometheus.client.stream.CKMSStream;
+import io.prometheus.client.stream.Quantile;
+import io.prometheus.client.stream.Stream;
+
+import java.util.*;
 
 /**
  * Summary metric, to track the size of events.
```

Comment:
Not sure by maybe you want to improve the documentation as well (Before sending the pull request to Prometheus)